### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by CalumTowers**

### CODE REVIEW

**Comment by AxonC** [comment]
What does this do?

**Comment by atoff** [comment]
> What does this do?

+1

**Comment by daviesnathan** [comment]
LGTM 👍

File: .travis.yml [hunk 1]
```diff
@@ -30,10 +30,11 @@ before_script:
 - php artisan serve -q &
 
 script:
-- vendor/bin/phpunit --coverage-clover=coverage.xml
+- if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then vendor/bin/phpunit; fi
```
Comment (atoff) [inline .travis.yml]: You spelt if wrong

**Comment by daviesnathan** [comment]
There isn't a JIRA ticket number in the PR name.

**Pull request closed without merging**